addons/hold-delete/addon.json

```diff
@@ -0,0 +1,58 @@
+{
+  "name": "Hold to delete sprites",
+  "description": "Require the delete button to be held down in order to delete a sprite.",
+  "tags": ["editor"],
+  "credits": [
+    {
+      "name": "DNin01",
+      "link": "https://github.com/DNin01"
+    }
+  ],
+  "userscripts": [
+    {
+      "url": "userscript.js",
+      "matches": ["projects"]
+    }
+  ],
+  "userstyles": [
+    {
+      "url": "hide-confirmation.css",
+      "matches": ["projects"],
+      "if": {
+        "settings": { "noConfirm": true }
+      }
+    }
+  ],
+  "settings": [
+    {
+      "name": "Hold duration",
+      "id": "duration",
+      "type": "select",
+      "potentialValues": [
+        {
+          "id": "short",
+          "name": "Short"
+        },
+        {
+          "id": "default",
+          "name": "Default"
+        },
+        {
+          "id": "long",
+          "name": "Long"
+        }
+      ],
+      "default": "default"
+    },
+    {
+      "name": "Skip confirmation",
+      "id": "noConfirm",
+      "type": "boolean",
+      "default": false
```

Comment:
```suggestion
      "default": true
```
I think the addon is much more useful if this setting is on by default, so I've come up with an idea for a notice to inform users that the press and hold gesture will skip the confirmation message:
```json
"info": [
  {
    "type": "notice",
    "id": "removesConfirmation",
    "text": "If the \"skip confirmation\" setting below is enabled, the hold gesture replaces the confirmation prompt that would appear after using the delete button."
  }
]
```

<details><summary>An alternative to that information...</summary>
<p>

...would be to rename the addon (but I don't know how well that will go):
```json
{
  "name": "Replace deletion confirmation with press and hold",
  "description": "Remove the confirmation prompt when deleting a sprite and instead require the delete button to be held down in order to delete the sprite."
}
```

</p></details>